src/button/component.jsx

```diff
@@ -187,6 +192,14 @@ export let Button : Component<ButtonOptions> = create({
             queryParam: true
         },
 
+        isFundingThrottleEnabled: {
+            type:     'boolean',
+            required: false,
+            def() : boolean {
+                return false;
```

Comment:
Can we set up the throttle here? `def()` will be passed props, so you should be able to do

```
def(props) {
    fundingLogoThrottle = buildFundingLogoThrottle(props);
    return fundingLogoThrottle && fundingLogoThrottle.isEnabled();
}
```

This is better than overwriting `props.isFundingThrottleEnabled = true;` and means that the correct value will be sent to the server, which we'll need to do for the server-side portion of the render to work correctly.

Reply:
Looks like the funding decorate function is called first, so I'm setting up the throttle in that block, but then using it to define the isFundingThrottleEnabled in the def() above.

Reply:
Hmm this is going to be problematic... lemme circle back on this one.